### PR TITLE
New Arch_Linux Cheat Sheet

### PR DESCRIPTION
**Pull request by direwolf424**
**What does your Instant Answer do?**
It is a cheat sheet for Pacman which is the default Package Manager for Arch Linux.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Every distro consist of its own package manager and so different commands . thus a cheat sheet is neccesary.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://wiki.archlinux.org/index.php/Pacman

**Why did you choose this data source?**
It is the official page for Arch Linux.

**Are there any other alternative (better) data sources?**
Man Page of Pacman and Yaourt

**What are some example queries that trigger this Instant Answer?**
arch linux cheat sheet , pacman cheat sheet , yaourt cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Arch users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**
The Documentation was very good . I want to contribute more often . 

**Where did you hear about DuckDuckHack? (For first time contributors)**
Quora

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 10
        - [X] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 11
        - [] Microsoft edge
        
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Linux
        - [X] Google Chrome
        - [] Firefox
        
    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

-----
IA Page: https://duck.co/ia/view/arch_linux_cheat_sheet

### CODE REVIEW

**Comment by direwolf-424** [comment]
Both Changes done . Plz review it

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                "key": "find xyz",
+                "val": "Search for a file xyx"
+            },
+            {
+                "key": "mount /dev/sdx /media/xyz",
+                "val": "Mount the partition sdx to xyz directory"
+            },
+            {
+                "key": "killall xyz",
+                "val": "kill all running instances of a process xyz"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/arch-linux.json]: @direwolf-424 correct  the casing

Comment (direwolf-424) [inline share/goodie/cheat_sheets/json/arch-linux.json]: thnks fr the info . its done .

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
+{
+    "id": "arch_linux_cheat_sheet",
+    "name": "Arch Linux",
+    "description": "Arch Linux Commands For Pacman , Yaourt and Some Basic Linux Commands",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/arch-linux.json]: nit: leading space before the `,` should be removed

**Comment by moollaza** [comment]
@direwolf424 this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@direwolf424 By the way, we think you did a great job with this! Are you interested in helping others who build similar Instant Answers in the future?

If yes, we'll reach out and add you to a private list of mentors as well as send over some DDG merch as an additional thank you. Just email <open@duckduckgo.com> with the subject 'I want to help.' Please include your shipping address, phone-number (required for shipping outside USA) and t-shirt size. We'll reach out from there!

Otherwise, thank you for this amazing contribution. We hope you'll keep exploring and playing around with DuckDuckHack.

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com
                
                mobile: https://ia-screenshots.s3.amazonaws.com/arch_linux_cheat_sheet_mobile.png

**Comment by moollaza** [comment]
@direwolf424 congrats! It's live: https://duckduckgo.com/?q=arch+linux+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**